### PR TITLE
Run periodic jobs only twice a day on weekends

### PR DESCRIPTION
**Pull request by kit1980**

### CODE REVIEW

File: .github/workflows/periodic.yml [hunk 1]
```diff
@@ -2,8 +2,10 @@ name: periodic
 
 on:
   schedule:
-    - cron: 45 0,8,16 * * *
-    - cron: 45 4,12,20 * * *
+    - cron: 45 0,8,16 * * 1-5
+    - cron: 45 0 * * 0,6
+    - cron: 45 4,12,20 * * 1-5
+    - cron: 45 12 * * 0,6
```
Comment (ZainRizvi) [inline .github/workflows/periodic.yml:8]: We have a lot of schedules now :)
How about adding a comment explaining the what/why over here

**Review by ZainRizvi — APPROVED** [review]

**Review by malfet — APPROVED** [review]

File: .github/workflows/periodic.yml (hunk 1 above)
Comment (kit1980) [reply .github/workflows/periodic.yml]: Added comments.

File: .github/workflows/periodic.yml [hunk 2]
```diff
@@ -2,8 +2,10 @@ name: periodic
 
 on:
   schedule:
-    - cron: 45 0,8,16 * * *
-    - cron: 45 4,12,20 * * *
+    - cron: 45 0,8,16 * * 1-5
+    - cron: 45 0 * * 0,6
```
[the remark this answers is not in the document]
Comment (kit1980) [reply .github/workflows/periodic.yml]: Changed to 9:45pm PST Fri and Sat evening.

**Pull request closed without merging**